src/applications/personalization/rated-disabilities/tests/e2e/rated-disabilities.cypress.spec.js

```diff
@@ -0,0 +1,61 @@
+import disableFTUXModals from 'platform/user/tests/disableFTUXModals';
+import mockDisabilities from '../mockdata/200-response.json';
+import mockTotalRating from '../mockdata/total-rating-response.json';
+import mockErrorResponse from '../mockdata/error-response.json';
+
+const RATED_DISABILITIES_PATH = '/disability/view-disability-rating/rating';
+const DISABILITIES_ENDPOINT =
+  '/disability_compensation_form/rated_disabilities';
+const TOTAL_RATING_ENDPOINT = '/disability_compensation_form/rating_info';
+
+const testAxe = () => {
+  cy.injectAxe();
+  cy.axeCheck();
+};
+
+const testHappyPath = () => {
+  cy.intercept('GET', DISABILITIES_ENDPOINT, mockDisabilities).as(
+    'mockDisabilities',
+  );
+  cy.intercept('GET', TOTAL_RATING_ENDPOINT, mockTotalRating).as(
+    'mockTotalRating',
+  );
+  cy.wait(['@mockDisabilities', '@mockTotalRating']);
```

Comment:
This wait isn't needed. The next line will still pass

Reply:
Might also be worth doing any axe check immediately, before APIs have resolved

Reply:
Thanks for the heads up about the `cy.wait` 👍 . Can you explain the piece about the axeCheck a little further? Not sure I understand what the benefit is.

Reply:
Ah sure. In this case it would be checking the DOM for violations while the loading spinner was present. Ideally we should check the DOM for violations _every time it changes_. I started doing this after Josh found some issues with my work during his a11y review. I was doing axe scans, but didn't run in a particular scenario that _did_ have violations. When I started adding in more aXe scans I discovered that the `LoadingButton` is problematic in its `loading` state if you don't give it a `loadingText` prop.

Reply:
so to be totally clear: I meant to do an aXe scan while data is loading. And do another after data has loaded and the page is showing its real contents.